en/advanced/best-practice-security.md

```diff
@@ -152,6 +153,16 @@ app.use(session({
 }))
 ```
 
+## Prevent brute-force attacks against authorisation
+
+Make sure login endpoints are protected to make private data more secure.
+
+One simple and in the same time quite powerful way is to block authorisation attempts by 2 metrics:
+1. The first is number of consecutive failed attempts by Username and IP pair. 
+1. The second is number of failed attempts from IP per long period of time. For example, block IP on 100 failed attempts per day.
+
+[rate-limiter-flexible](https://github.com/animir/node-rate-limiter-flexible) package provides tools to make that easy and fast. You can find [an example of brute-force protection in docs](https://github.com/animir/node-rate-limiter-flexible/wiki/Overall-example#login-endpoint-protection)
```

Comment:
The [rate-limiter-flexible](https://github.com/animir/node-rate-limiter-flexible) package provides tools to make this technique easy and fast. You can find [an example of brute-force protection in the documentation](https://github.com/animir/node-rate-limiter-flexible/wiki/Overall-example#login-endpoint-protection).